OpenDreamRuntime/DreamManager.cs

```diff
@@ -204,12 +216,12 @@ internal partial class DreamManager : IDreamManager {
                 throw new NotImplementedException($"Ref for {value} is unimplemented");
             }
 
-            // The first digit is the type, i.e. 1 for objects and 2 for strings
-            return $"{(int) refType}{idx}";
+            // The first digit is the type
+            return $"[0x{((int) refType+idx):X}]";
```

Comment:
BYOND gives these in lowercase
```suggestion
            return $"[0x{((int) refType+idx):x}]";
```